tests/xtro-sharpie/common-Network.ignore

```diff
@@ -0,0 +1,28 @@
+# the following protocols ae binded manualy as NativeObject
+
+!missing-protocol! OS_nw_advertise_descriptor not bound
+!missing-protocol! OS_nw_connection not bound
+!missing-protocol! OS_nw_content_context not bound
+!missing-protocol! OS_nw_endpoint not bound
+!missing-protocol! OS_nw_error not bound
+!missing-protocol! OS_nw_interface not bound
+!missing-protocol! OS_nw_listener not bound
+!missing-protocol! OS_nw_parameters not bound
+!missing-protocol! OS_nw_path not bound
+!missing-protocol! OS_nw_path_monitor not bound
+!missing-protocol! OS_nw_protocol_definition not bound
+!missing-protocol! OS_nw_protocol_metadata not bound
+!missing-protocol! OS_nw_protocol_options not bound
+!missing-protocol! OS_nw_protocol_stack not bound
+
+# binded but with a name alias changes and xtro gets confused.
+!missing-pinvoke! nw_ip_create_metadata is not bound
+!missing-pinvoke! nw_udp_create_metadata is not bound
```

Comment:
I don't follow your comment ?!? where's xtro being confused.
AFAICS that code is under `#if false` (in `src/Network/NWProtocolMetadata.cs`) so xtro is right to report them as missing

Reply:
True, sorry, I did not see the #if, I'll update the comment to state the rdar.